samples/nrf9160/modem_shell/src/cloud/cloud_lwm2m.c

```diff
@@ -227,6 +243,10 @@ static void cloud_lwm2m_rd_client_event_cb(struct lwm2m_ctx *client_ctx,
 	case LWM2M_RD_CLIENT_EVENT_REGISTRATION_COMPLETE:
 		mosh_print("LwM2M: Registration complete");
 		connected = true;
+		/* Check if session lifetime needs to be updated. */
+		if (update_session_lifetime) {
```

Comment:
The code is fine now. Maybe it would be nice to have a comment on why session lifetime is updated conditionally and not always.

Reply:
I'll add a comment in `LWM2M_RD_CLIENT_EVENT_BOOTSTRAP_REG_COMPLETE` event handling where the flag is set.